docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -1,143 +1,72 @@
+// application code, <<synthetics-test-suite,use a test suite>>.
+// * If you want to create a single journey and manage it individually, <<synthetics-inline-journey,use an inline monitor>>.
 
 [discrete]
-[[synthetics-test-suite]]
-== Use a test suite
-
-A test suite can contain any number of journeys in any number of files using the file extensions `.journey.ts` or `.journey.js`.
-You can use the `@elastic/synthetics` package as a library to run a test suite.
-
-To test a test suite locally, start by writing your tests:
-
-. Create a new https://docs.npmjs.com/cli/v7/commands/npm-init[NPM/Node.js project].
-. Create a `javascript` or `typescript` file that imports your tests (using the `.journey.ts` or `.journey.js` file extension).
-. Make sure any dependencies are defined with an `import` outside of the `journey` object.
-. Compile everything together.
-
-See an example of a test suite in the https://github.com/elastic/synthetics-demo[elastic/synthetics-demo] repository.
-The examples below show how to test that repository's `todo` directory locally.
-
-Clone the repository, change into the directory containing the synthetics test suite,
-and install the necessary dependencies including `@elastic/synthetics`:
-
-[source,sh]
-----
-# Check out the synthetics repo and included examples
-git clone git@github.com:elastic/synthetics-demo.git &&\
-cd synthetics-demo/todos/ &&\
-# Install all required dependencies for the todos example
-npm install
-----
-
-[NOTE]
-==============
-If you want to test your own suite locally, be sure to install the `@elastic/synthetics` package:
-
-[source,sh]
-----
-npm install -g @elastic/synthetics
-----
-==============
-
-From inside the directory containing the synthetics test suite, run the provided tests.
-By default, the `elastic-synthetics` runner will only run files matching the filename `*.journey.(ts|js)*`.
-
-[source,sh]
-----
-# Run tests on the current directory. The dot `.` indicates
-# that it should run all tests in the current directory.
-npx @elastic/synthetics .
-----
+[[monitor-use]]
+== Configure monitor
 
-[discrete]
-[[synthetics-test-suite-send]]
-=== Send to {stack}
+When you're ready to create a monitor to run your tests, you'll need to set a few configuration options:
 
-After testing locally, you can run the test suite using the Elastic Synthetics integration or {heartbeat}
-depending on <<uptime-set-up-choose,the approach you chose to set up>>.
+* **Give your monitor a name.** Provide a human readable name for the monitor.
+This will appear in {kib} where you can view and manage monitors after they're created.
+* **Set the schedule.** Specify the interval at which your tests will run.
+* **Specify where to find the tests.** This will vary based on the method you use to create a montior.
+* **Set other options as needed.** There are several other options you can set to customize your implementation including params, tags, screenshot options, throttling options, and more.
```

Comment:
This is a high level overview of configuration options across available implementation methods.